### PR TITLE
Contribute a table and text recognition demo with a GUI.

### PR DESCRIPTION
**Pull request by WenjingKangIntel**
Contribute a table and text recognition demo with a GUI.

This demo workflow mainly has two parts:

The first part is the detection of the table. This part uses OpenCV to perform perspective transformation, and then detects the cells of the table.

The second part uses the cell images detected in the first part for handwritten text recognition with OpenVINO. It first reads a cell image and performs the preprocessing such as further binarization, resize and padding, and then the inference will start. After decoding the returned indexes into characters, the demo will display the predicted text. Optionally, you can generate the predicted text into a docx file.

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/handwritten_table_text_recognition_demo/python/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Ubuntu 18.04 or above.
+
+### OpenVINO Requirement
+
+You should install OpenVINO Toolkit (version >= 2020.3).
+
+For more OpenVINO toolkit installation information, please refer to [here](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html).
+
+### Other Dependencies
+This project requires Python 3.7.
```
Comment (eaidova) [inline demos/handwritten_table_text_recognition_demo/python/README.md:29]: openvino has minimal requirements for python is 3.6
Do you use some python3.7 specific things in the demo?
Could you please relax dependencies on python to give more openvino users run your demo?

File: demos/handwritten_table_text_recognition_demo/python/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+This demo workflow mainly has two parts:
+
+- The first part is the detection of the table. This part uses OpenCV to perform perspective transformation, and then detects the cells of the table.
+
+- The second part uses the cell images detected in the first part for handwritten text recognition with OpenVINO. It first reads a cell image and performs the preprocessing such as further binarization, resize and padding, and then the inference will start. After decoding the returned indexes into characters, the demo will display the predicted text. Optionally, you can generate the predicted text into a docx file.
+
+## Installation and dependencies
+### OS Requirement
```
Comment (eaidova) [inline demos/handwritten_table_text_recognition_demo/python/README.md:19]: is there some platform specific behavior or you just test it using this OS?
I mean is it really should not work on Mac OS and Windows?

File: demos/handwritten_table_text_recognition_demo/python/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+### OpenVINO Requirement
+
+You should install OpenVINO Toolkit (version >= 2020.3).
+
+For more OpenVINO toolkit installation information, please refer to [here](https://docs.openvinotoolkit.org/latest/openvino_docs_install_guides_installing_openvino_linux.html).
+
+### Other Dependencies
+This project requires Python 3.7.
+
+Required python modules：
```
Comment (eaidova) [inline demos/handwritten_table_text_recognition_demo/python/README.md:31]: usually we provide demo requirements in the file `requirements.in`, like here
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/3d_segmentation_demo/python/requirements.txt

File: demos/handwritten_table_text_recognition_demo/python/README.md (hunk 2 above)
Comment (cocokale) [reply demos/handwritten_table_text_recognition_demo/python/README.md]: It has **only** been tested in Ubuntu. It should work in Mac OS and windows.

File: demos/handwritten_table_text_recognition_demo/python/README.md (hunk 1 above)
Comment (cocokale) [reply demos/handwritten_table_text_recognition_demo/python/README.md]: Yes. I've tested this demo in pyton3.6. It worked. This requirement can be relaxed to python3.6.

File: demos/handwritten_table_text_recognition_demo/python/README.md (hunk 3 above)
Comment (cocokale) [reply demos/handwritten_table_text_recognition_demo/python/README.md]: Delete the requirements in README, just put them in the requirement.txt.

**Pull request closed without merging**